docs/sources/tempo/metrics-generator/_index.md

```diff
@@ -46,3 +46,9 @@ Dimensions can be the service name, the operation, the span kind, the status cod
 The more dimensions are enabled, the higher the cardinality of the generated metrics.
 
 To learn more about this processor, read the [documentation]({{< relref "span_metrics/" >}}).
+
+### Exporting metrics
+
+The metrics-generator runs a Prometheus Agent that periodically sends metrics to a remote_write endpoint.
+The remote_write endpoint is configurable and can be any Prometheus-compatible endpoint.
+In order to support multi-tenancy, the metrics-generator forwards the `X-Scope-OrgID` header to the remote_write endpoint.
```

Comment:
I added links to the configuration file and to the Prometheus read_write endpoint (which is linked in the Config file).

```suggestion
The metrics-generator runs a Prometheus Agent that periodically sends metrics to a `remote_write` endpoint.
The `remote_write` endpoint is configurable and can be any [Prometheus-compatible endpoint](https://prometheus.io/docs/prometheus/latest/configuration/configuration/#remote_write).
To learn more about the endpoint configuration, refer to the [Metrics-generator]({{< relref "../configuration/#metrics-generator" >}}) section of the Tempo Configuration documentation. 

To support multi-tenancy, the metrics-generator forwards the `X-Scope-OrgID` header to the remote_write endpoint.
```